Resources/Prototypes/Entities/Items/Consumables/drinks.yml

```diff
@@ -647,7 +628,7 @@
     sprite: Objects/Drinks/glass_brown.rsi
 
 - type: entity
-  parent: DrinkBase
+ parent: DrinkBase
```

Comment:
@FL-OZ You have a few of these in the file, just in case you didn't spot the others.

Reply:
ah, rip